saleor/webhook/tests/test_webhook_payloads.py

```diff
@@ -103,7 +112,6 @@ def test_generate_order_payload(
     assert payload["collection_point_name"] is None
     assert payload.get("token") == order_with_lines.token
     assert payload.get("shipping_method")
-    assert payload.get("shipping_tax_rate")
```

Comment:
Is it expected?

Reply:
Yes, we're testing it in `test_generate_order_payload_prices`, as it isn't a field that will be in both taxed and untaxed payloads.